docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,47 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?from=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``from`` and ``limit`` are required only for pagination.
+By default, a ``limit`` of 100 is used.
+The parameter ``user_id`` can be used to filter by user id using an SQL regexp.
```

Comment:
regular expressions use syntax like `.` to mean "any character" and `.*` to mean "any number of any character".

I think this is a "SQL wildcard":

```suggestion
The parameter ``user_id`` can be used to filter by user id using an SQL wildcard.
```

Reply:
The SQL wildcards are actually just `%` or `_`, but some implementations like MS also use `*` and `?`. Futhermore I don't want to suggest to use those placeholders, the minimum this attribute should support with any database is a search for the given token.